src/analyses/variable-sensitivity/variable_sensitivity_domain.h

```diff
@@ -80,26 +80,28 @@
   "(vsd-unions):"                                                              \
   "(vsd-flow-insensitive)"                                                     \
   "(vsd-data-dependencies)"                                                    \
-  "(vsd-liveness)"                                                             \
-                                                                               \
-    // clang-format off
+  "(vsd-liveness)"
```

Comment:
Are we still doing the "put a \ at the end of even the last line so that new additions are just 1-line diffs" thing?